test/tmt/main.fmf

```diff
@@ -0,0 +1,33 @@
+# Only common dependencies that are NOT required to run netavark-tests.sh are
+# specified here. Everything else is in netavark-tests.sh.
+require:
+    - cargo
+    - make
+
+adjust:
+    duration: 10m
+    when: arch == aarch64
+
+/validate:
+    tag: upstream
+    summary: Validate test
+    test: make -C ../.. validate
+
+/unit:
+    tag: upstream
+    summary: Unit test
+    test: make -C ../.. unit
+
+/upstream:
+    tag: upstream
+    summary: Run tests on upstream PRs
+    
```

Comment:
This and line 31 are trailing whitespace. I'm a little alarmed that the git hooks didn't block it.

Reply:
If you're talking about pre-commit hooks, I don't see any setup on this repo :disappointed:

Reply:
removed